mRemoteV1/UI/Forms/frmMain.cs

```diff
@@ -961,6 +977,106 @@ private void SetQuickConnectProtocol(string protocol)
 		}
         #endregion
 
+        #region Multi SSH
```

Comment:
Rather than adding this as a region, let's make this its own class. That way we can reuse it elsewhere if needed and frmMain wont get any bigger